sources/host-ctr/cmd/host-ctr/main.go

```diff
@@ -288,6 +288,20 @@ func runCtr(containerdSocket string, namespace string, containerID string, sourc
 			oci.WithHostNamespace(runtimespec.NetworkNamespace),
 			oci.WithHostHostsFile,
 			oci.WithHostResolvconf,
+			// Unmask `/sys/firmware` by passing an alternate list of masked paths
+			// List is based on the DefaultUnixSpec's MaskedPaths for Linux
+			// (https://github.com/containerd/containerd/blob/main/oci/spec.go)
```

Comment:
nit: a permalink to the exact `MaskedPaths` list would be preferable, in case that list gets refactored out of the file